### ISSUE TITLE
Accessibility testing for Compatibility Tool

### ISSUE THREAD

**Issue by kienstra** [ampproject/amp-wp#1591]
Please test the Compatibility Tool's 4 screens for accessibility:

# [AMP Validation Error Index](https://native-ampconfdemo.pantheonsite.io/wp-admin/edit-tags.php?taxonomy=amp_validation_error&post_type=amp_validated_url) (#1361):
<img width="1188" alt="error-index-screen" src="https://user-images.githubusercontent.com/4063887/48033084-8536d880-e11f-11e8-817e-9c6455228a6c.png">

# [AMP Validated URLs](https://native-ampconfdemo.pantheonsite.io/wp-admin/edit.php?post_type=amp_validated_url) (#1362):
<img width="1189" alt="url-index-screen" src="https://user-images.githubusercontent.com/4063887/48033150-c8914700-e11f-11e8-84e7-cbc4f3e54fec.png">

# [Single Error](https://native-ampconfdemo.pantheonsite.io/wp-admin/edit.php?amp_validation_error=8c40891607dd8d1201507fc3b1368f7e&post_type=amp_validated_url) (#1364):
<img width="1171" alt="single-error-screen" src="https://user-images.githubusercontent.com/4063887/48033042-5fa9cf00-e11f-11e8-8d54-269329e5692a.png">

# [Single URL](https://native-ampconfdemo.pantheonsite.io/wp-admin/post.php?post=514&action=edit) (#1365):
<img width="1140" alt="single-url-screen" src="https://user-images.githubusercontent.com/4063887/48032946-06da3680-e11f-11e8-8cc3-73dd28f6b629.png">

**Comment by kienstra** [comment]
**Admin Pointer 'Tooltips'**

Thanks to some [good input](https://github.com/Automattic/amp-wp/issues/1400#issuecomment-424232988), we know the admin pointer 'tooltips' on these screens aren't accessible (#1479).

**Comment by kienstra** [comment]
**Reporting Issues**

Feel free to open a separate issue for each of the 4 screens if you'd like, or raise all of them in this issue if they fit.

**Comment by westonruter** [comment]
Should be done as part of #2316.

**Comment by kmyram** [comment]
Solved in #2316
